src/files/test_file_reading.py

```diff
@@ -40,9 +40,10 @@ def test_files_open():
     that the file is properly closed after its suite finishes, even if an exception is raised at
     some point. Using with is also much shorter than writing equivalent try-finally blocks:
     """
+    path = 'src/files/'
 
     # Open files without using 'with' statement.
-    file = open('src/files/multi_line_file.txt', 'r')
+    file = open(path + 'multi_line_file.txt', 'r')
```

Comment:
How about we do this file path generation with `.format`? Could you try it? You might need to update the path declaration.

Reply:
done